packages/frontend-api/tests/Unit/Model/Token/TokenFacadeTest.php

```diff
@@ -27,12 +28,13 @@
 class TokenFacadeTest extends TestCase
 {
     /**
-     * @dataProvider tokensDataProvider
-     * @param \Lcobucci\JWT\UnencryptedToken $token
+     * @param array $tokenData
      * @param class-string|null $exceptionClass
      */
-    public function testTokenValidation(UnencryptedToken $token, ?string $exceptionClass): void
+    #[DataProvider('tokensDataProvider')]
+    public function testTokenValidation(array $tokenData, ?string $exceptionClass): void
     {
+        $token = $this->createToken($tokenData['issuedBy'], $tokenData['privateKey'], $tokenData['expiresAt']);
```

Comment:
it would be nice to have these keys in class constants

Reply:
I have expanded it to separate method parameters